### PR TITLE
Toggle seek bar visibility on tap and INFO button press

### PR DESCRIPTION
**Pull request by andrew-shulgin**
Also, reset hide timeout when a touch has been handled.

### CODE REVIEW

File: app/src/main/java/is/xyz/mpv/TouchGestures.kt [hunk 1]
```diff
@@ -114,6 +114,6 @@ class TouchGestures(val width: Float, val height: Float, val observer: TouchGest
             }
         }
 
-        return true
+        return false
```
Comment (sfan5) [inline app/src/main/java/is/xyz/mpv/TouchGestures.kt]: doesn't `false` mean the event was not handeled?

Comment (andrew-shulgin) [reply app/src/main/java/is/xyz/mpv/TouchGestures.kt]: You're right. I erroneously thought this return statement is reached only if no gesture has been handled.
I'll fix this.

Comment (andrew-shulgin) [reply app/src/main/java/is/xyz/mpv/TouchGestures.kt]: There are two options:

- Remove onKeyDown from MPVActivity and handle tap with TouchGestures as a PropertyChange;
- Return false and leave the tap handler in MPVActivity.

What do you think is better?

Comment (sfan5) [reply app/src/main/java/is/xyz/mpv/TouchGestures.kt]: latter seems much cleaner to me

**Pull request merged**